packages/vue3/src/components/Divider/Divider.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import type { Orientation, Style } from './types';
+
+withDefaults(
+  defineProps<{
+    orientation?: Orientation
+    borderStyle?: Style
+  }>(),
+  {
+    orientation: 'horizontal',
+    borderStyle: 'solid',
+  },
+);
+</script>
+
+<template>
+  <div :class="[orientation, borderStyle]" />
+</template>
+
+<style scoped lang="scss">
+.horizontal {
+  width: 100%;
+  border-width: 1px;
+  border-color: var(--gray-300);
```

Comment:
`gray-100` seems to be the most subtle looking, let's use it 🙏 
```suggestion
  border-color: var(--gray-100);
```